tests/ext/root_span_url_with_query_params.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
```

Comment:
This test was just renamed, but another test was created with the same name this one used to have so it appears as new instead...